src/liapunovspectrum.jl

```diff
@@ -98,26 +98,28 @@ function liap_jetcoeffs!(eqsdiff!, t::Taylor1{T}, x::Vector{Taylor1{U}},
         dx::Vector{Taylor1{U}}, xaux::Vector{Taylor1{U}},
         δx::Array{TaylorN{Taylor1{U}},1}, dδx::Array{TaylorN{Taylor1{U}},1},
         jac::Array{Taylor1{U},2}) where {T<:Real, U<:Number}
-
     order = x[1].order
-
     # Dimensions of phase-space: dof
     nx = length(x)
     dof = round(Int, (-1+sqrt(1+4*nx))/2)
-
     for ord in 0:order-1
         ordnext = ord+1
 
         # Set `taux`, auxiliary Taylor1 variable to order `ord`
-        taux = Taylor1(t.coeffs[1:ordnext])
+        @inbounds taux = Taylor1( t.coeffs[1:ordnext] )
         # Set `xaux`, auxiliary vector of Taylor1 to order `ord`
         for j in eachindex(x)
             @inbounds xaux[j] = Taylor1( x[j].coeffs[1:ordnext] )
         end
 
         # Equations of motion
-        eqsdiff!(taux, xaux, dx)
-        stabilitymatrix!( eqsdiff!, t[0], view(xaux,1:dof), δx, dδx, jac )
+        for ind in eachindex(δx)
+            @inbounds δx[ind] = xaux[ind] + _δv[ind]
+        end
+        eqsdiff!(taux, δx, dδx)
+        @inbounds dx[1:dof] .= constant_term.(dδx)
+        # Stability matrix
```

Comment:
I do see why this approach leads to some improvement; you avoid certain allocations in `stabilitymatrix!` related to the independent variables that are represented by `_δv` now. I think it would be nice to still have `stabilitymatrix!` being called, maybe by passing to it `_δv` as a new argument, but avoiding allocations the way you do it.

Reply:
Also, we were evaluating twice the equations of motion per timestep per order; with this change now we are evaluating only once the equations of motion per timestep per order.

Reply:
Nice!